frontend/src/components/pages/ManageStoryTranslationPage.tsx

```diff
@@ -1,22 +1,241 @@
+  };
+
+  const callSoftDeleteStoryTranslationMutation = async () => {
+    await deleteStoryTranslation({
+      variables: {
+        id: storyTranslationIdParam,
+      },
+    });
+    closeModal();
+    history.push("/");
+  };
+
+  const filterStyle = useStyleConfig("Filter");
   return (
-    <Box textAlign="center">
-      <h1>
-        Manage story translation {storyIdParam} {storyTranslationIdParam} :)
-      </h1>
-    </Box>
+    <Flex direction="column" height="100vh">
+      <Header title="Manage Story Translation" />
+      <Flex direction="row" flex={1}>
+        <Flex sx={filterStyle} maxWidth="292px">
+          <Heading size="lg">{title}</Heading>
+          <Heading size="sm">Story ID</Heading>
+          <Text>{storyIdParam}</Text>
+          <Heading size="sm">Translation Language</Heading>
+          <Text>{convertLanguageTitleCase(language)}</Text>
+          <Heading size="sm">Level</Heading>
+          <Text>{level}</Text>
+          <Heading size="sm">Stage</Heading>
+          <Text>{convertStageTitleCase(stage)}</Text>
+          {/* TODO: Use translatorName and reviewerName */}
+          <Heading size="sm">Translator</Heading>
+          <Link isExternal href={`/user/${translatorId}`}>
+            {translatorId}
+            {translatorName}
+          </Link>
+          {reviewerId && (
+            <>
+              <Heading size="sm">Reviewer</Heading>
+              <Link isExternal href={`/user/${reviewerId}`}>
+                {reviewerId}
+                {reviewerName}
+              </Link>
+            </>
+          )}
+        </Flex>
+        <Flex direction="column" margin="40px" width="50%">
+          <FormControl id="email">
+            <Heading size="sm" marginTop="24px" marginBottom="18px">
+              Title
+            </Heading>
+            <Input type="title" value={title || ""} />
+            <Heading size="sm" marginTop="24px" marginBottom="18px">
+              Description
+            </Heading>
+            <Textarea type="description" value={description || ""} />
+            <Heading size="sm" marginTop="24px" marginBottom="18px">
+              YouTube Link
+            </Heading>
+            <Input type="youtubeLink" value={youtubeLink} />
+          </FormControl>
+          <Heading size="sm" marginTop="24px">
+            Delete Story Translatoin
+          </Heading>
+          <Text>
+            Permanently delete this story translation and all data associated
+            with it.
+          </Text>
+          <Button
+            colorScheme="red"
+            margin="10px 0px"
+            width="250px"
+            onClick={() => openModal()}
+          >
+            Delete Story translation
+          </Button>
+        </Flex>
+        <Box
+          css={{
+            borderRadius: "8px",
+            margin: "40px",
+            MozBorderRadius: "8px",
+            overflow: "hidden",
+            webkitBorderRadius: "8px",
+          }}
```

Comment:
Nit: for me this box is too tall so the bottom two corners of the iframe are not rounded.